src/backend/distributed/commands/table.c

```diff
@@ -1299,20 +1296,22 @@ ConvertPostgresLocalTablesToCitusLocalTables(AlterTableStmt *alterTableStatement
 		else if (IsCitusTableType(relationId, CITUS_LOCAL_TABLE))
 		{
 			CitusTableCacheEntry *entry = GetCitusTableCacheEntry(relationId);
-			if (!entry->autoConverted || autoConverted)
+			if (!entry->autoConverted)
```

Comment:
this change is only refactor for ease of read. Do you agree that this simplifies the code?